projects/organization-management/src/app/store/users/users.reducer.ts

```diff
@@ -21,13 +21,18 @@ const initialState: UsersState = usersAdapter.getInitialState({
 
 export function usersReducer(state = initialState, action: UsersAction): UsersState {
   switch (action.type) {
+    case UsersActionTypes.AddUser:
```

Comment:
reducer tests are missing

Reply:
the original branch came without the reducer.spec. Maybe its easier to add the tests for the actions i used after merging everything right?

Reply:
yes, you can skip the tests